flaml/searcher/blendsearch.py

```diff
@@ -734,6 +736,12 @@ def suggest(self, trial_id: str) -> Optional[Dict]:
                 result = {self._metric: reward, self.cost_attr: 1, "config": config}
                 self.on_trial_complete(trial_id, result)
                 return None
+        if self._use_incumbent_result_in_evaluation:
+            if self._trial_proposed_by[trial_id] > 0:
+                choice_thread = self._search_thread_pool[
+                    self._trial_proposed_by[trial_id]
+                ]
+                config["INCUMBENT_RESULT"] = choice_thread.best_result

```

Comment:
To clarify, I don't mean to use "INCUMBENT_RESULT" as the actual key. I meant to define a constant variable `INCUMBENT_RESULT = "__incumbent_result__"` and use the variable instead of the actual string everywhere. That way we can easily change the string value. See some examples in data.py.